pipenv/vendor/pep517/meta.py

```diff
@@ -9,7 +9,7 @@
 try:
     import importlib.metadata as imp_meta
 except ImportError:
-    import pipenv.vendor.importlib_metadata as imp_meta
+    import importlib_metadata as imp_meta
```

Comment:
We never fail here with ImportError. But it's still confusing that this was changed.

Reply:
I think I recall pondering about it because `import pipenv.vendor.importlib_metadata` doesn't exist -- It was confusing to me we would reference a fallabck import that does not exist.